robot_description : 'robot_description'
base_link : 'base'
floor_to_base_height : -0.9

#default grasp params
joints : ['right_gripper_1_finger_joint']
pregrasp_posture : [-0.0125]
pregrasp_time_from_start : 0.0
grasp_posture : [0.0095]
grasp_time_from_start : 0.0


#desired pose from end effector to grasp
grasp_pose_to_eef : [-0.15, 0.0, 0.0]

end_effector_name: 'right_hand'
end_effector_parent_link: 'right_wrist'

